### PR TITLE
Changes report preparing generator

### PR DESCRIPTION
**Pull request by Trodrige**
Removing TCMA files, fixed bug in patient list creation report, and made report menu more explicit

### CODE REVIEW

File: sites/default/menu_data.json [hunk 1]
```diff
@@ -590,7 +590,7 @@
             "children" : [ ],
             "requirement" : 0
         }, { 
-            "label" : "Appt-Enc",
+            "label" : "Appointments and Encounters",
```
Comment (aethelwulffe) [inline sites/default/menu_data.json]: This one is really a financial report....  Question is, what should we view it as doing?  Appointments vs. Encounters?  This name is good enough until the real reports come in I suppose!

File: sites/default/menu_data.json [hunk 2]
```diff
@@ -625,23 +625,23 @@
             "children" : [ ],
             "requirement" : 0
         }, { 
-            "label" : "Chart Activity",
+            "label" : "Chart Location Activity",
```
Comment (aethelwulffe) [inline sites/default/menu_data.json]: Paper Chart Tracking?

Comment (aethelwulffe) [reply sites/default/menu_data.json]: Never known anyone to actually use this...

File: sites/default/menu_data.json [hunk 3]
```diff
@@ -625,23 +625,23 @@
...
             "menu_id" : "rep0",
             "target" : "rep",
             "url" : "/interface/reports/chart_location_activity.php",
             "children" : [ ],
             "requirement" : 0,
             "global_req" : "!disable_chart_tracker"
         }, { 
-            "label" : "Charts Out",
+            "label" : "Charts Checked Out",
```
Comment (aethelwulffe) [inline sites/default/menu_data.json]: Or this

File: interface/reports/reports_controllers/AdjustmentsController.php [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+		   	echo $adj_name . '
+		  			</td>
+		  			<td align="right">';
+		   				echo oeFormatMoney($total);
+		 	  echo '</td>
+		 		  </tr>';
+
+        } // end not export
+  	}
+}
```
Comment (teryhill) [inline interface/reports/reports_controllers/AdjustmentsController.php]: Art will git you on the needed linefeed.

File: sites/default/menu_data.json [hunk 5]
```diff
@@ -514,7 +514,7 @@
     "children" : [ { 
         "label" : "Clients",
         "children" : [ { 
-            "label" : "List",
+            "label" : "Patient List",
```
[the remark this answers is not in the document]
Comment (Trodrige) [reply sites/default/menu_data.json]: I'll rename this next commit.

File: interface/reports/reports_controllers/AdjustmentsController.php (hunk 4 above)
Comment (Trodrige) [reply interface/reports/reports_controllers/AdjustmentsController.php]: handled!

**Comment by teryhill** [comment]
@Trodrige are you ready for this to be merged?

File: sites/default/menu_data.json (hunk 3 above)
Comment (Trodrige) [reply sites/default/menu_data.json]: The near future will decide the fate of these so called reports.

**Comment by Trodrige** [comment]
@teryhill I'll merge this now. I'm currently trying to separate PHP code from HTML
Will be doing a pull request to this branch, so you see the changes.

**Pull request merged**